src/main/java/net/irisshaders/iris/api/v0/item/IrisItemColorProvider.java

```diff
@@ -0,0 +1,14 @@
+package net.irisshaders.iris.api.v0.item;
+
+import net.coderbot.iris.vendored.joml.Vector3f;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+
+public interface IrisItemColorProvider
```

Comment:
This shouldn't be a separate interface Imo

Reply:
Sure, I can merge it into `IrisItemLightProvider` if it's a better fit

Reply:
Not sure if this should use JOML's Vector3f. This makes it hard for other mods to support without a second dependency, while MC's Vec3 would work fine.